presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -835,6 +837,48 @@ public static Slice normalize(@SqlType("varchar(x)") Slice slice, @SqlType("varc
         return utf8Slice(Normalizer.normalize(slice.toStringUtf8(), targetForm));
     }
 
+    @Description("labels whether input strings use Unicode or Zawgyi font encoding")
+    @ScalarFunction
+    @SqlNullable
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice myanmarFontEncoding(@SqlType(StandardTypes.VARCHAR) Slice slice)
+    {
+        if (slice == null) {
```

Comment:
You don't need to handle null. By default if a parameter is not annotated with @SqlNullable null value will not invoke the function.